.github/workflows/release.yml

```diff
@@ -53,22 +53,21 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Push Operator Container image to GHCR
-        uses: docker/build-push-action@v2
-        with:
-          context: .
-          file: dask_kubernetes/operator/deployment/Dockerfile
-          push: ${{ github.repository == 'dask/dask-kubernetes' && github.event_name == 'push' && startsWith(github.ref, 'refs/tags') }}
-          platforms: linux/amd64
-          tags: ghcr.io/dask/dask-kubernetes-operator:latest,ghcr.io/dask/dask-kubernetes-operator:${{ steps.get_version.outputs.VERSION }}
-
       - name: Build and publish operator Helm chart with chartpress
-        if: github.repository == 'dask/dask-kubernetes' && github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
         env:
           # chartpress can make use of a personal access token by setting these
           # environment variables like this, for details see:
           # https://github.com/jupyterhub/chartpress/blob/d4e2346d50f0724f6bee387f4f8aebc108afb648/chartpress.py#L118-L128
           #
           GITHUB_ACTOR: ""
           GITHUB_TOKEN: "${{ secrets.dask_bot_token }}"
-        run: cd dask_kubernetes/operator/deployment/helm && chartpress --publish-chart
+          PUBLISH: ${{ github.repository == 'dask/dask-kubernetes' && github.event_name == 'push' && startsWith(github.ref, 'refs/tags') }}
+        run: |
+          cd dask_kubernetes/operator/deployment/helm
+
+          if ${PUBLISH} == true; then
+              chartpress --force-push --force-publish-chart
```

Comment:
Yeah I think there are a couple of different points here. I added the `if` statement to only push/publish on tags so that chartpress builds the docker images for us (previously we only ran the chartpress step on tags).

I also changed the flags to the `--force-*` flavours because we occasionally have to delete and recreate tags on release and that would be necessary to have the chart overwritten.